pkg/virt-handler/non-root.go

```diff
@@ -124,7 +124,7 @@ func getTapDevices(vmi *v1.VirtualMachineInstance, networkBindings map[string]v1
 		_, isTapNetwork := tapNetworks[net.Name]
 		if podInterfaceName, exists := networkNameScheme[net.Name]; isTapNetwork && exists {
 			tapDevices[net.Name] = podInterfaceName
-		} else if isTapNetwork && !exists {
+		} else if isTapNetwork {
```

Comment:
Put `isTapNetwork` into an if-clause around the `podInterfaceName` if clause?

E.g.

```
if _, isTapNetwork := tapNetworks[net.Name]; isTapNetwork {
...
}
```

Reply:
It's a problem since `isTapNetwork` is used in both `if` and `else if`

Reply:
Readability enhancement:
```golang
		if _, isTapNetwork := tapNetworks[net.Name]; !isTapNetwork {
			continue
		}
		if podInterfaceName, exists := networkNameScheme[net.Name]; exists {
			tapDevices[net.Name] = podInterfaceName
		} else {
			return nil, fmt.Errorf("network %q not found in naming scheme: this should never happen", net.Name)
		}
```
OR
```golang
		if _, isTapNetwork := tapNetworks[net.Name]; !isTapNetwork {
			continue
		}
		
		if _, exists := networkNameScheme[net.Name]; !exists {
			return nil, fmt.Errorf("network %q not found in naming scheme: this should never happen", net.Name)
		}
		
		tapDevices[net.Name] = networkNameScheme[net.Name]
```